br/pkg/lightning/restore/table_restore.go

```diff
@@ -722,6 +722,9 @@ func (tr *TableRestore) postProcess(
 				}
 
 				remoteChecksum, err := DoChecksum(ctx, tr.tableInfo)
+				if err != nil {
```

Comment:
Can we have https://github.com/kisielk/errcheck linter enabled for this kind of bug?

Reply:
Seems the `errcheck` linter could not check this kind of bug.